assets/scss/_variables_project.scss

```diff
@@ -1,46 +1,37 @@
-/* docsy-delta full file override: we're not tracking changes to the docsy file of the same name. */
+/* Docsy-delta full file override: we're not tracking changes to the Docsy file of the same name. */
 
-$twitter-blue: #00aced;
-$gitter-magenta: #ec1363;
-$opentelemetry-orange: #f5a800;
-$opentelemetry-orange-light: #fff0ce;
-$opentelemetry-blue: #4f62ad;
-$opentelemetry-blue-light: #cee1ff;
-$opentelemetry-purple: #425cc7;
-$light-green: #c0f587;
+$otel-colors: (
+  'orange': #f5a800,
+  'orange-light': #fff0ce,
+  'blue': #4f62ad,
+  'blue-light': #cee1ff,
+  'purple': #425cc7,
+  'light-green': #c0f587,
+);
 
-$primary: $opentelemetry-blue;
-$secondary: $opentelemetry-orange;
-$td-enable-google-fonts: false;
-$google_font_name: 'Noto Sans';
-$google_font_family: 'Noto+Sans:300,400,600,700';
```

Comment:
Redefining `$google_font_*` has no effect when `$td-enable-google-fonts` is false